src/rust/ensogl/lib/core/src/display/style/sheet.rs

```diff
@@ -155,8 +155,12 @@ impl Debug for Expression {
 
 impl PartialEq for Expression {
     fn eq(&self, other:&Self) -> bool {
-        let same_args     = self.args == other.args;
-        let same_function = Rc::ptr_eq(&self.function,&other.function);
+        let same_args = self.args == other.args;
+        // The *const u8 cast discards the vtable to work around
+        // https://github.com/rust-lang/rust/issues/46139
```

Comment:
Please extend the comment, so reading the comment would be enough to understand whats happening here. Lets use links only as "learn more" - the comment should be enough to get first grasp off whats going on.